src/Outsystems.SetupTools/Lib/PlatformSetup.ps1

```diff
@@ -431,6 +442,16 @@ function InstallDotNetCore([string]$Sources)
     return $($result.ExitCode)
 }
 
+function InstallDotNetCore([string]$Sources)
+{
+    return InnerInstallDotNetCore -InstallVersion21 $False -Sources $SourcePath
```

Comment:
This should be $Sources and not $SourcePath

Reply:
Fixed